loom/graphics/gfxVectorGraphics.cpp

```diff
@@ -170,7 +170,6 @@ void VectorGraphics::endFill() {
 void VectorGraphics::moveTo(float x, float y) {
     auto path = getPath();
     path->moveTo(x, y);
-    inflateBounds(Loom2D::Rectangle(x, y, 0, 0));
```

Comment:
Ah, what happens if you `moveTo(-50, -50)` and then `lineTo(100, 100)`? Does it still pick up that the bounds are then `-50, -50, 150, 150`?